packages/webapp/public/locales/en/translation.json

```
@@ -1894,7 +1894,11 @@
     "TODO": "To do",
     "TRANSPLANT": "Transplant",
     "TRANSPLANT_LOCATIONS": "Select a transplant location",
-    "UNASSIGNED": "Unassigned"
+    "UNASSIGNED": "Unassigned",
+    "CREATE": {
+      "FAILED": "Can't create the task",
+      "LOCATION_DELETED": "It seems you are trying to create a task on a deleted location. Please try again afresh!"
```

I'd suggest

```suggestion
      "LOCATION_DELETED": "Another farmer has retired your selected location. Please refresh the page to get the most recent status."
```
so it's more consistent with a pre-existing string we have for attempting to delete a plan with completed tasks.